storage/umass/umass.c

```diff
@@ -667,7 +667,7 @@ static void umass_poolthr(void *arg)
 		while (umass_common.rqueue == NULL) {
 			condWait(umass_common.rcond, umass_common.rlock, 0);
 		}
-		req = umass_common.rqueue->prev;
+		req = umass_common.rqueue;
```

Comment:
OK, but the FIFO behavior still depends on the list implementation adding a new element to the tail. Please add a comment explaining this.